### PR TITLE
Zimperium MTD Connector

### PR DESCRIPTION
**Pull request by zcarsco**
Fixes #
- Zimperium Mobile Threat Defense connector, initial commit

### CODE REVIEW

File: DataConnectors/Zimperium MTD Alerts.json [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+                    "delete": true
+                }
+            }
+        ],
+        "customs":""
+    },
+    "instructionSteps": [
+        {
+            "title": "Configure and connect Zimperium MTD",
+            "description": "1. In zConsole, click **Manage** on the navigation bar.\n2. Click the **Integrations** tab.\n3. Click the **Threat Reporting** button and then the **Add Integrations** button.\n4. Create the Integration:\n  - From the available integrations, select Microsoft Azure Sentinel.\n  - Enter your workspace id and primary key from the fields below, click **Next**.\n  - Fill in a name for your Azure Sentinel integration.\n  - Select a Filter Level for the threat data you wish to push to Azure Sentinel.\n  - Click **Finish**",
```
Comment (preetikr) [inline DataConnectors/Zimperium MTD Alerts.json]: Please add a link to the Zimperium documentation on zConsole specifics.

File: DataConnectors/Zimperium MTD Connector.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# Connect your Zimperium Mobile Threat Defense to Azure Sentinel
```
Comment (preetikr) [inline DataConnectors/Zimperium MTD Connector.md]: Remove the md file from this PR and please email me the md file - The documentation GitHub repo is different and not in this repo

Comment (zcarsco) [reply DataConnectors/Zimperium MTD Connector.md]: I have removed that file and sent via email as requested.

File: DataConnectors/Zimperium MTD Alerts.json (hunk 1 above)
Comment (zcarsco) [reply DataConnectors/Zimperium MTD Alerts.json]: Added link to Connector file.

**Pull request merged**